src/wallet/wallet.cpp

```diff
@@ -2896,7 +2896,7 @@ void CWallet::ListAvailableSigmaMintCoins(vector<COutput> &vCoins, bool fOnlyCon
     LOCK2(cs_main, cs_wallet);
     list<CSigmaEntry> listOwnCoins;
     CWalletDB walletdb(pwalletMain->strWalletFile);
-    listOwnCoins = zwalletMain->GetTracker().MintsAsZerocoinEntries();
```

Comment:
Could you explain what is mature sigma mint?

Reply:
A mint with at least 6 confs.